src/drivers/camera_trigger/camera_trigger.cpp

```diff
@@ -69,18 +71,26 @@
 #include "interfaces/src/pwm.h"
 #include "interfaces/src/seagull_map2.h"
 
-#define TRIGGER_PIN_DEFAULT 1
-
 extern "C" __EXPORT int camera_trigger_main(int argc, char *argv[]);
 
-typedef enum : int32_t {
+typedef enum : uint8_t {
 	CAMERA_INTERFACE_MODE_NONE = 0,
 	CAMERA_INTERFACE_MODE_GPIO,
 	CAMERA_INTERFACE_MODE_SEAGULL_MAP2_PWM,
 	CAMERA_INTERFACE_MODE_MAVLINK,
 	CAMERA_INTERFACE_MODE_GENERIC_PWM
 } camera_interface_mode_t;
 
+typedef enum : uint8_t {
+	TRIGGER_MODE_NONE = 0,
+	TRIGGER_MODE_INTERVAL_ON_CMD,
+	TRIGGER_MODE_INTERVAL_ALWAYS_ON,
+	TRIGGER_MODE_DISTANCE_ALWAYS_ON,
+	TRIGGER_MODE_DISTANCE_ON_CMD
+} trigger_mode_t;
+
+#define commandParamToInt(n) static_cast<int>(n >= 0 ? n + 0.5f : n - 0.5f)
```

Comment:
Always hesitant to use those on NuttX. Is it safe? Used anywhere else in the system?